icu4c/source/common/uhash.h

```diff
@@ -128,7 +128,7 @@ typedef UElementsAreEqual UValueComparator;
 /* see cmemory.h for UObjectDeleter and uprv_deleteUObject() */
 
 /**
- * This specifies whether or not, and how, the hastable resizes itself.
```

Comment:
The rest of this file uses "hashtable" in many places. I think it might be better to change this "hashtable" in order to be consistent with the rest of the file.